main/app.yaml

```diff
@@ -0,0 +1,19 @@
+module: default
```

Comment:
I looked at some other repos but wasn't able to find examples with licensing info in them. I'll take a look tomorrow morning at the actual requirement.